javascripts/main.js

```diff
@@ -41,8 +41,30 @@
     }).val(labels).trigger('chosen:updated').change(function (e) {
       location.href = updateQueryStringParameter(getFilterUrl(), 'labels', encodeURIComponent(($(this).val() || "")));
     });
+
+    projectsPanel.find("ul.popular-tags").children().each(function(i, elem){
+        $(elem).on("click", function(){
+            let selTags = ($('.tags-filter').val() || [])
+            const selectedTag = preparePopTagName($(this).text() || "");
```

Comment:
Hi @juliankrieger - This will fail on IE

![testing-on-ie](https://user-images.githubusercontent.com/18662248/50091937-6561fc80-0215-11e9-8327-677026b52e36.png)